recipe/meta.yaml

```diff
@@ -1,31 +1,29 @@
 {% set name = "Shapely" %}
-{% set version = "1.8.4" %}
+{% set version = "2.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version.split('.post')[0] }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a195e51caafa218291f2cbaa3fef69fd3353c93ec4b65b2a4722c4cf40c3198c
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6250e79ca299222a239ce44df613d228f1216c5cc31351217bf377b03d7be4e0
 
 build:
   number: 0
   # geos currently isn't available on s390x
-  skip: True  # [py<36 or s390x]
+  skip: True  # [py<37 or s390x]
```

Comment:
```suggestion
  skip: True  # [py<37 or s390x]
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

Reply:
Got a new error after adding the script: 
`Building wheel for shapely (pyproject.toml): finished with status 'error'
  ERROR: Failed building wheel for shapely
Failed to build shapely
ERROR: Could not build wheels for shapely, which is required to install pyproject.toml-based projects`